megatron/training.py

```diff
@@ -964,6 +964,7 @@ def add_to_logging(name):
             total_loss_dict[nan_iters_key])
         log_string += ' samples per second: {:.3f} |'.format(samples_per_sec)
         log_string += ' TFLOPs: {:.2f} |'.format(tflops)
+        log_string += ' Model Params (B): {:.1f} |'.format(approx_parameters_in_billions)
```

Comment:
I guess this will print every log iteration? Maybe we can just print it once at the start?

Reply:
Updated to print just before start of training